src/dctest/core.cljs

```diff
@@ -320,6 +320,14 @@ Options:
 ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
 ;; Main
 
+(defn js-process []
+  (let [process (select-keys (js->map js/process)
+                             ["platform" "pid" "ppid" "argv"
+                              "versions" "features"])
+        env (into {} (js->map js/process.env))]
+    (merge (merge process
```

Comment:
This is _so_ merged ;-)

Reply:
Haha. Sometimes structural refactoring results in some humorous droppings like that. Will fix